src/envoy/http/jwt_auth/token_extractor.cc

```diff
@@ -82,9 +82,18 @@ void JwtTokenExtractor::Extract(
   for (const auto& header_it : header_maps_) {
     const HeaderEntry* entry = headers.get(header_it.first);
     if (entry) {
+      std::string token;
+      absl::string_view val = entry->value().getStringView();
+      if (val.find(" ") != absl::string_view::npos) {
```

Comment:
I think you can simplify this by using absl::StrSplit and take the last element from its result.

Reply:
or use `rfind`, then substr(pos)

Reply:
thanks, updated.

Reply:
It is not simplified at all... at least cache the find result so don't do it twice.

Reply:
address this comment in https://github.com/istio/proxy/pull/2075